### ISSUE TITLE
Add a wiki: Plazma Burst Wiki

### ISSUE THREAD

**Issue by KevinPayravi** [KevinPayravi/indie-wiki-buddy#551]
An anonymous user has submitted a new wiki via our [submission form](https://form.getindie.wiki).

Link to origin wiki: https://plazmabursttwo.fandom.com
Link to destination (independent) wiki: https://plazmaburst.miraheze.org

**Comment by KevinPayravi** [comment]
Concerned about Miraheze wiki's lack of activity, accessibility issues, and lack of maintenance (e.g. broken widget on main page).
